src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: 48px;
```

Comment:
do not set external geometry on a block level. margin-right should be part of main_card element or something like it